charts/refarch-template/charts/backend/values.yaml

```diff
@@ -0,0 +1,99 @@
+replicaCount: 1
+
+image:
+  registry: ghcr.io
+  repository: it-at-m/refarch/refarch-backend
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+
+podAnnotations: {}
+podLabels: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
```

Comment:
das kann man auch aktivieren

Reply:
@hupling siehe oben

Reply:
siehe https://github.com/it-at-m/helm-charts/pull/40#discussion_r1744858116

Reply:
wie oben: kommentieren.